src/repart/repart.c

```diff
@@ -550,6 +570,20 @@ static const char *encrypt_mode_table[_ENCRYPT_MODE_MAX] = {
         [ENCRYPT_KEY_FILE_TPM2] = "key-file+tpm2",
 };
 
+static const char *integrity_mode_table[_INTEGRITY_MODE_MAX] = {
+        [INTEGRITY_OFF] = "off",       /* no integrity protection */
+        [INTEGRITY_INLINE] = "inline", /* luks2 storage when encrypted, hardware sector integrity fields otherwise */
+        [INTEGRITY_DATA] = "data",     /* interleave data and integrity tags on the same device */
+        [INTEGRITY_META] = "meta",     /* use a separate device for storing integrity tags */
```

Comment:
we nowadays want that such tables are aligned by the `=` character, see empty_mode_table[] above